chapters/en/chapter10/4.mdx

```diff
@@ -0,0 +1,343 @@
+        num_rows: 28539
+    })
+    validation: Dataset({
+        features: ['file', 'audio', 'text', 'speaker_id', 'chapter_id', 'id'],
+        num_rows: 2703
+    })
+    test: Dataset({
+        features: ['file', 'audio', 'text', 'speaker_id', 'chapter_id', 'id'],
+        num_rows: 2620
+    })
+})
+```
+Great! We have our training, validation and test splits ready, with 28539, 2703 and 2620 samples respectively. The two data columns `audio` and `text` contain the most important information for our task: the raw audio inputs and the target text outputs.
+
+Let’s take a look at the first example of the train split:
+```python
+raw_datasets["train"][0]
+```
+
+```
+{'file': '/home/sanchit_huggingface_co/.cache/huggingface/datasets/downloads/extracted/d2da1969fe9e7d06661b5dc370cf2e3c119a14c35950045bcb76243b264e4f01/374-180298-0000.flac',
+ 'audio': {'path': '/home/sanchit_huggingface_co/.cache/huggingface/datasets/downloads/extracted/d2da1969fe9e7d06661b5dc370cf2e3c119a14c35950045bcb76243b264e4f01/374-180298-0000.flac',
+  'array': array([ 7.01904297e-04,  7.32421875e-04,  7.32421875e-04, ...,
+         -2.74658203e-04, -1.83105469e-04, -3.05175781e-05]),
+  'sampling_rate': 16000},
+ 'text': 'CHAPTER SIXTEEN I MIGHT HAVE TOLD YOU OF THE BEGINNING OF THIS LIAISON IN A FEW LINES BUT I WANTED YOU TO SEE EVERY STEP BY WHICH WE CAME I TO AGREE TO WHATEVER MARGUERITE WISHED',
+ 'speaker_id': 374,
+ 'chapter_id': 180298,
+ 'id': '374-180298-0000'}
+```
+
+Alright! We can see that we have our target text sample ready - it looks very much like it's taken from an audiobook! But what on earth is going on with the audio? 🤔
+
+As it turns out, we represent audio data digitally as a 1-dimensional array. Each array value denotes the amplitude of our audio signal at a particular time step. From the amplitude information, we can reconstruct the frequency spectrum of the audio and recover all the acoustic features.
+
+Since our audio input is continuous, we can't represent an amplitude value for every possible time step. Instead, we _sample_ amplitude values at fixed time steps.
+
+The interval with which we sample our audio is known as the _sampling rate_. For our dataset, we can see that the sampling rate is 16000, meaning 16000 amplitude values are provided each second. Keep the sampling rate in the back of your mind, it'll be important when we come to processing our data later on!
+
+Whilst 1-dimensional arrays are a suitable format for machines, they're not much use to us as humans! We want to be able to **listen** to our audio to get a feel for the speech and recording conditions. Here, we have two options:
+
+1. Convert our 1-dimensional array into a human-friendly format (mp3 or wav).
+2. Look to the Hugging Face Hub!
+
+Option 2 seems much easier to me! Let's head over to the LibriSpeech ASR dataset card on the Hugging Face Hub: https://huggingface.co/datasets/librispeech_asr
```

Comment:
I suggest having a screenshot here to guide the reader - great idea with the audio widget!

(Alternatively, we could use a Gradio Space and embed that directly)